src/Io/PsrAwaitRequestHandler.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace FrameworkX\Io;
+
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use React\Http\Message\Response;
+use function React\Async\await;
+use function React\Promise\resolve;
+
+/**
+ * @internal
+ */
+class PsrAwaitRequestHandler implements RequestHandlerInterface
+{
+
+    /**
+     * @var callable|null
+     */
+    private $next;
+
+    public function __construct(callable $next = null) {
+        $this->next = $next;
+    }
+
+    public function handle(ServerRequestInterface $request): ResponseInterface
+    {
+        if ($this->next === null) {
+            return new Response();
+        }
```

Comment:
This is a bit weird. But there are at least two occurrences of a handler invocation without the `next` parameter in the framework:
* [\FrameworkX\Io\RouteHandler::__invoke](https://github.com/clue/framework-x/blob/bd2c75897b2551d1afa91767561c0cf7db11594b/src/Io/RouteHandler.php#L105)
* [\FrameworkX\Container::callable](https://github.com/clue/framework-x/blob/bd2c75897b2551d1afa91767561c0cf7db11594b/src/Container.php#L101)

and this was the easiest solution I could think of to avoid these from exploding :)